From 5bdcd3d30e8a714da306dac348d39aaff3143067 Mon Sep 17 00:00:00 2001
From: Amelia Crate <amelia.crate@chainguard.dev>
Date: Thu, 13 Nov 2025 10:56:01 -0600
Subject: [PATCH] Fix broken upstream presets

These only work if one builds with the new repo[1], and knows how to correlate
a release of the old repo to the commit in the new repo it should build with.
Only upstream knows how to do that, and they don't publish that correlation
anywhere.
See upstream issue[2] regarding the fact that it's currently confusing and sometimes
impossible to correctly package this as a downstream consumer.

[1] https://github.com/GoogleCloudPlatform/google-guest-agent
[2] https://github.com/GoogleCloudPlatform/guest-agent/issues/574
---
 90-google-guest-agent.preset | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/90-google-guest-agent.preset b/90-google-guest-agent.preset
index b0da932..365b131 100644
--- a/90-google-guest-agent.preset
+++ b/90-google-guest-agent.preset
@@ -1,6 +1,6 @@
-disable google-guest-agent.service
+enable google-guest-agent.service
 enable google-shutdown-scripts.service
 enable google-startup-scripts.service
-disable gce-workload-cert-refresh.timer
-enable google-guest-compat-manager.service
-enable google-guest-agent-manager.service
\ No newline at end of file
+enable gce-workload-cert-refresh.timer
+disable google-guest-compat-manager.service
+disable google-guest-agent-manager.service
-- 
2.50.1

